src/components/RoomHeaderAvatars.js

```diff
@@ -45,13 +47,18 @@ const RoomHeaderAvatars = (props) => {
     return (
         <View pointerEvents="none">
             <View style={[styles.flexRow, styles.wAuto, styles.ml3]}>
-                {_.map(iconsToDisplay, (val, index) => (
-                    <View key={`${val}${index}`} style={[styles.justifyContentCenter, styles.alignItemsCenter]}>
+                {_.map(iconsToDisplay, (icon, index) => (
+                    <View key={`${icon.source}${index}`} style={[styles.justifyContentCenter, styles.alignItemsCenter]}>
```

Comment:
Using source as key cause a regression https://github.com/Expensify/App/issues/28074

Reply:
Thanks for the note @narefyev91.